projects/swimlane/ngx-ui/src/lib/components/dialog/dialog.component.html

```diff
@@ -1,20 +1,27 @@
 <div [class]="class" [class.ngx-dialog]="true" [style.zIndex]="zIndex">
   <div
     class="ngx-dialog-content {{ cssClass }}"
+    [class.large]="format === 'large'"
```

Comment:
Question: should we be insulating nix-UI classes from accidentally being classes within other codebases? Since large is a common word and potentially a css class elsewhere we could accidentally have style override each other without view encapsulation. 


Curious if it's a concern is all.

Reply:
makes sense.